lms/djangoapps/instructor/enrollment.py

```diff
@@ -495,7 +495,7 @@ def send_mail_to_student(student, param_dict, language=None):
 
     message_class = ace_emails_dict[message_type]
     message = message_class().personalize(
-        recipient=Recipient(username='', email_address=student),
+        recipient=Recipient(lms_user_id=0, email_address=student),
```

Comment:
what's the result of using lms_user_id=0?

Reply:
Well, it would depend on the configured ACE backend. Only backend that cares about it would be the new Braze backend. In which case if it receives a zero, it will fall back to Django email sending.

But that doesn't really even matter. Since the three places here that don't have a user id are all transactional emails. And those are configured to send via the django email channel directly, never even hitting the braze backend, which would fall back to django email anyway if it did get there (ACE has two channel settings - one channel for transactional emails and one for marketing ones - in edx prod, we use django_email and sailthru_email for those)